app/views/layouts/homepage.html.erb

```diff
@@ -0,0 +1,23 @@
+<% content_for(:navbar) do %>
+  <nav class="navbar navbar-default navbar-static-top" role="navigation">
```

Comment:
`<%= render '/controls' %>` can be used here instead of the `nav` block.

Reply:
Updated.